specs/flashtestations.md

```diff
@@ -468,33 +456,21 @@ This operation returns the raw attestation quote that was used to register the T
 
 1. **Simple Storage Model**: The registry maintains a simple mapping between TEE-controlled addresses, workloadIds, and their attestation quotes without tracking complex endorsement relationships.
 
-2. **Individual Verification**: Instead of batch removal based on endorsement bundles, attestations are verified individually when needed, allowing for more granular management.
-
-3. **Quote Storage**: The system maintains a copy of the attestation quote used to register each address, supporting external verification and auditability.
+2. **Quote Storage**: The system maintains a copy of the attestation quote used to register each address, supporting external verification and auditability.
 
-4. **Gas Efficiency**: The lookup operation must be extremely efficient (O(1) gas costs) regardless of the number of addresses stored.
+3. **Gas Efficiency**: The lookup operation must be extremely efficient (O(1) gas costs) regardless of the number of addresses stored.
 
-5. **Reliance on Off-Chain Observation (Initial Version)**: In its current design, the protocol relies on an honest off-chain service (e.g., operated by TEE operators or other trusted entities) to monitor attestations and their associated collateral. This service is expected to call `verifyAttestation` to mark an attestation as stale if the underlying collateral is updated or becomes invalid. This implies a trust assumption in these off-chain entities to maintain the integrity of the registry's "valid" attestations.
+4. **Reliance on Off-Chain Observation (Initial Version)**: In its current design, the protocol relies on an honest off-chain service (e.g., operated by the TEE infrastructure operators or other trusted entities) to monitor attestations and their associated collateral. This service is expected to call `verifyAttestation` to mark an attestation as stale if the underlying collateral is updated or becomes invalid. This implies a trust assumption in these off-chain entities to maintain the integrity of the registry's "valid" attestations.
```

Comment:
verifyAttestation -> reverifyAttestation

Reply:
There are a couple other places in this spec where we didn't update verifyAttestation -> reverifyAttestation and I can't comment directly on them with GitHub. Please update them as well